packages/components/src/base-control/stories/index.tsx

```diff
@@ -60,6 +60,7 @@ WithHelpText.args = {
 export const WithVisualLabel: ComponentStory< typeof BaseControl > = (
 	props
 ) => {
+	// @ts-expect-error - Unclear how to fix, see also https://github.com/WordPress/gutenberg/pull/39468#discussion_r827150516
 	BaseControl.VisualLabel.displayName = 'BaseControl.VisualLabel';
```

Comment:
Side effect of the changing the TS types for VisualLabel! We might want to look into a solution if we encounter this more.